scripts/add.tokens.erc20.mjs

```diff
@@ -0,0 +1,377 @@
+
+export const ${testnetToken}: RequiredErc20Token = {
+	id: SEPOLIA_${symbol}_TOKEN_ID,
+	network: SEPOLIA_NETWORK,
+	standard: 'erc20',
+	category: 'default',
+	name: '${name}',
+	symbol: '${symbol}',
+	decimals: ${symbol}_DECIMALS,
+	icon: ${icon},
+	address: '${testnetContractAddress}',
+	exchange: 'erc20',
+	twinTokenSymbol: 'ckSepolia${symbol}'
+};
+`
+			: '';
+
+	if (newFileContentMainnet + newFileContentTestnet === '') {
+		console.log(`No new variables to create for token ${symbol}`);
+		return { fileName, mainnetToken: undefined, testnetToken: undefined };
+	}
+
+	const newFileContent =
+		fileContentFirstPartWithImports +
+		fileContentDecimalsConst +
+		newFileContentMainnet +
+		newFileContentTestnet;
+	writeFileSync(filePath, newFileContent);
+	filesCreatedOrModified = true;
+
+	if (newFileContentMainnet !== '') {
+		console.log(`Created new mainnet variables for token ${symbol} in file ${filePath}`);
+	}
+
+	if (newFileContentTestnet !== '') {
+		console.log(`Created new testnet variables for token ${symbol} in file ${filePath}`);
+	}
+
+	return {
+		fileName,
+		mainnetToken: newFileContentMainnet !== '' ? mainnetToken : undefined,
+		testnetToken: newFileContentTestnet !== '' ? testnetToken : undefined
+	};
+};
+
+const updateTokensErc20Env = ({ fileName, mainnetToken, testnetToken }) => {
+	const filePath = path.join(DATA_DIR_PATH, 'tokens.erc20.env.ts');
+	let content = readFileSync(filePath, 'utf8');
+
+	const regexList = [
+		{
+			regex: /const ERC20_TWIN_TOKENS_MAINNET: RequiredErc20Token\[] = \[([\s\S]*?)];/,
+			token: mainnetToken
+		},
+		{
+			regex: /const ERC20_TWIN_TOKENS_SEPOLIA: RequiredErc20Token\[] = \[([\s\S]*?)];/,
+			token: testnetToken
+		}
+	];
+
+	regexList
+		.filter(({ token }) => nonNullish(token))
+		.forEach(({ regex, token }) => {
+			const { content: newContent, hasChanged } = addElementListInContent({
+				content,
+				regex,
+				element: token
+			});
+			if (hasChanged) {
+				content = newContent;
+				console.log(`Included ${token} in token list in ${filePath}`);
+			}
+		});
+
+	if (nonNullish(mainnetToken) || nonNullish(testnetToken)) {
+		const imports = [mainnetToken, testnetToken].filter(Boolean);
+		const { content: newContent, hasChanged } = updateImportsInContent({
+			content,
+			imports,
+			module: `$env/${fileName.replace('.ts', '')}`
+		});
+		if (hasChanged) {
+			content = newContent;
+			console.log(`Added import statement for ${imports.join(', ')} in ${filePath}`);
+		}
+	}
+
+	writeFileSync(filePath, content);
+};
+
+const flattenData = (data) => {
+	return Object.keys(data).map((symbol) => ({
+		symbol,
+		...data[symbol]
+	}));
+};
+
+const flattenEnvironmentData = (data) => {
+	return Object.entries(data).reduce(
+		(acc, [environment, values]) => ({
+			...acc,
+			[environment]: flattenData(values)
+		}),
+		{}
+	);
+};
+
+const readSupportedTokens = () => {
+	const jsonPath = path.resolve(DATA_DIR_PATH, 'tokens.ckerc20.json');
```

Comment:
You can use `resolve`  instead of `path.resolve` given that you are already `import { resolve } from ...`

Reply:
done